lib/public/views/Runs/Details/RunDetailsModel.js

```diff
@@ -333,14 +315,11 @@ export class RunDetailsModel extends Observable {
 class RunDetailsTabbedPanelModel extends TabbedPanelModel {
     /**
      * Constructor
-     *
-     * @param {number} runNumber the run number of the current run
-     * @param {string|null|undefined} [panelKey=null] the current panel key (null will use default)
      */
-    constructor(runNumber, panelKey) {
+    constructor() {
         super(Object.values(RUN_DETAILS_PANELS_KEYS));
-        this._runNumber = runNumber;
-        this.currentPanelKey = panelKey;
+        this._runNumber = null;
+        this.currentPanelKey = 'logs';
```

Comment:
Never use magic strings, you always have constants. Probably that the currentPanelKey is already defining default panelKey at this point, else you should either not define it at all or set it to null? What was the original behavior?

Reply:
refactored

Reply:
Please read again my comment, and check how the code works!